<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardGame</class>
 <widget class="QMainWindow" name="CardGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CardGame</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
background-color: #FFFFFF;
border: 1px solid black;
padding: 5px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="tries">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color: #FFFFFF;
border: 1px solid black;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>No. of Tries Remaining</string>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color: #FFFFFF;
border: 1px solid black;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>Score</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
background-color: #FFFFFF;
border: 1px solid black;
padding: 5px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newGame">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #FFFFFF;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>481</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_16">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_23">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_17">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="pushButton_18">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_14">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="pushButton_30">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_20">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_28">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_21">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="pushButton_24">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_19">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_15">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_22">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="pushButton_29">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="pushButton_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_26">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_27">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_25">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_13">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pushButton_12">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #ADD8E6;
border: 1px solid black;
padding: 5px;
}
QPushButton:pressed{
background-color: #C0C0C0;
border: 1px solid black;
padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>?</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
